ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -440,6 +468,98 @@ export default {
         }
       },
     },
+    {
+      id: 'E12S Promise Chiseled Sculpture Tether',
+      netRegex: NetRegexes.tether({ target: 'Chiseled Sculpture', id: '0011' }),
+      promise: async (data, matches) => {
+        // Calculate distance to center to determine inner vs outer
+        data.statues = data.statues || [];
+        if (data.statues.filter((s) => s.id === matches.targetId).length === 0) {
+          // Push statue to array immediately or else async causes the array to get duplicates of the statues
+          const statue = {
+            id: matches.targetId,
+          };
+          data.statues.push(statue);
+          const statueId = parseInt(matches.targetId, 16);
+          const statueData = await window.callOverlayHandler({
+            call: 'getCombatants',
+            ids: [statueId],
+          });
+
+          if (statueData === null || !statueData.combatants || !statueData.combatants.length)
+            return;
+
+          // Center = 0, -75
+          const x = statueData.combatants[0].PosX;
+          const y = statueData.combatants[0].PosY - -75;
+          statue.dist = Math.hypot(x, y);
+          if (data.statues.length > 1) {
+            // Sort so that closest statue (inner) is first
+            data.statues.sort((a, b) => a.dist - b.dist);
+          }
+        }
+      },
+      durationSeconds: 20,
+      infoText: (data, matches, output) => {
+        // Player is the source of the tether for this one, oddly enough
+        if (data.me !== matches.source)
+          return;
+
+        if (data.statues.length > 0 && data.statues[0].id === matches.targetId) {
+          data.statueDir = output.inner();
+        } else if (data.statues.length > 1 && data.statues[1].id === matches.targetId) {
+          data.statueDir = output.outer();
+        } else {
+          data.statueDir = '???';
+        }
+        return output.statue({ statueDir: data.statueDir });
+      },
+      outputStrings: {
```

Comment:
I think in this case where there are only two outputs, it's maybe easier for users / translators if it's just spelled out `Inner Statue` and `Outer Statue`.

Reply:
`data.statueDir` is stored off for later use in the `bait` triggers. I figured making it a separate string here would allow it to be modified for both locations easily.

Reply:
This is one of the things that outputstrings makes more complicated.  If you depend on the output string from one trigger in another, then it will be non-obvious to a user that they need to go change `E12S Promise Chiseled Sculpture Tether` in order to affect `E12S Promise Statue Laser First` and `E12S Promise Statue 2nd/3rd/4th Laser` text.

The usual pattern here is to store a key into outputStrings, e.g. store `'inner'` and `'outer'` in `data.statueDir` and then do `output[data.statueDir]()` .

Reply:
This should have been addressed in the latest updates, please confirm that I understood your intention correctly.

Reply:
:+1: